src/Helper/Html.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace eLife\Journal\Helper;
+
+final class Html
+{
+    private function __construct()
+    {
+    }
+
+    public static function stripElement(string $html, string $element) : string
+    {
+        // Assumes well-formed HTML.
+        return preg_replace("~</?{$element}(\s.*?)?>~i", '', $html);
```

Comment:
suspicious to use custom code and a regex to do this substitution, but if the subset of HTML is small enough it can work

Reply:
Indeed. Was hoping it would be done in JS as manipulating a DOM is far safer. Not replicating that here for performance.

This relies on our API producing safe, valid HTML, which we're already assuming elsewhere. (In this case, it relies on Annotations sanitising what comes from Hypothesis.)

Reply:
:+1: